### ISSUE TITLE
MME process has error status after successful attach with the orch8r and new Gateway

### ISSUE THREAD

**Issue by jrmejiaa** [magma/magma#2695]
### My Environment

- **Version:** `master tag v1.1.0`
- **Affected Component:** Access Gateway
- **Affected Subcomponent:** MME Service
- **Deployment Environment:** local docker-compose (orc8r), Openstack (AGW)

### Describe the Issue

Hi FB Team,

I have deployed Magma in the past, and I would like to create an automation process to create AGWx, which are going to be connected to the same orch8r and network, but with different gateways. However, I have a problem, because the MME does not work after a while. Everything goes well and there is no error, but the only service that does not up is the `magma@mme`. I have tracked the errors but I don't see any useful information.

```bash
● magma@mme.service - Magma OAI MME service
   Loaded: loaded (/etc/systemd/system/magma@mme.service; disabled; vendor preset: enabled)
   Active: activating (auto-restart) (Result: exit-code) since Wed 2020-09-09 20:43:07 UTC; 2s ago
  Process: 14636 ExecStopPost=/usr/bin/env python3 /usr/local/bin/service_util.py mme (code=exited, status=0/SUCCESS)
  Process: 14594 ExecStart=/usr/local/bin/mme -c /var/opt/magma/tmp/mme.conf -s /var/opt/magma/tmp/spgw.conf (code=exited, status=23)
  Process: 14573 ExecStartPre=/usr/bin/env python3 /usr/local/bin/generate_oai_config.py (code=exited, status=0/SUCCESS)
 Main PID: 14594 (code=exited, status=23)
```
Errors that I have seen in the logs: 

```bash
# One Error
==15305==ERROR: LeakSanitizer: detected memory leaks
# Second Error
Process: 5274 ExecStart=/usr/local/bin/mme -c /var/opt/magma/tmp/mme.conf -s /var/opt/magma/tmp/spgw.conf (code=killed, signal=SEGV)
# Third Error
ERROR:root:Error retrieving config for mme, key not found: use_stateless
ERROR:root:Error retrieving config for mme, key not found: enable_nat
# Fourth Error
Process: 12636 ExecStart=/usr/local/bin/mme -c /var/opt/magma/tmp/mme.conf -s /var/opt/magma/tmp/spgw.conf (code=killed, signal=ABRT)
```

**Expected behavior**
I would hope that using the python `addgw.py` and with a restart of the services, the Orchestrator start a connection with this new magma (new instance using OSM) with the new Gateway and I could connect later a enodeB. 

**Additional context**
Attach to this log and as a suggestion of @ssanadhya, I send the `/var/log/syslog`

[syslog_magma.txt](https://github.com/magma/magma/files/5198071/syslog_magma.txt)

**Comment by ssanadhya** [comment]
@jrmejiaa , thanks for sharing the syslog. The error ` GetAssignedIPBlock fails with code 14, msg: Connect Failed` indicates that the MME is crashing as it is unable to connect to Mobilityd. Can you re-try with the latest Magma version https://github.com/magma/magma/tree/v1.2.0? We have added retry attempts for the above "connect failed" issue.

Further, can you also share the config changes you made on Mobilityd as part of AGWx automation?

**Comment by jrmejiaa** [comment]
Hi @ssanadhya thanks for your answer. As soon as the `v1.2.0` was released I have installed a new version of the AGW and this problem was solved with the new version. I don't know why the connection was not successful with the previous version but with the same set up this time I could create a automation process using REST API and OSM. For those who are interested to do the same, it is necessary to **delete** the files `gateway.ctr` and `gateway.key`, which are located in `/var/opt/magma/certs`. 

If you does not delete those changes, the AGW is not going to updated the pool of the APN. The steps are straight forward in that case: 
* Change the Hardware ID
* Send the REST API `https://{ORCH_IP}:9443/magma/v1/lte/{ORCH_NET}/gateways` (See the Swagger to more information)
* Restart the service `sudo service magma@* restart`

Thanks for you help @ssanadhya
